### PR TITLE
make the EXODII sniper a reliable rifle

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
None

#### Purpose of change
EXODII Sniper being 5 shot only didn't make sense

#### Describe the solution
made it single shot and 2 shot burst. The two shot mentions it being a last ditch panic option as it really should be with such a massive cartridge. 

#### Testing
Looking good in game:
image

#### Additional context
image

image

### CODE REVIEW

**Comment by andrei8l** [comment]
Minor nitpick, but if the original was not an automatic weapon, its name would have been PS md. 71 (cf. real-life [PSL](https://en.wikipedia.org/wiki/PSL_(rifle))).

File: data/json/items/gun/exodii.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
       {
         "id": "pamd71z",
         "name": { "str": "PA md. 71z" },
```
Comment (bombasticSlacks) [inline data/json/items/gun/exodii.json:54]:
```suggestion
        "name": { "str": "PS md. 71z" },
```

**Comment by kevingranade** [comment]
You need to update the base item description too, you just updated the variant.

I would say drop the "sniper rifle" reference altogether, maybe, "to serve as a sniper weapon." -> "to serve as a dedicated large zombie hunting weapon.".

**Comment by I-am-Erk** [comment]
Calling it a hunting rifle would be fine. The idea is that the exodii use it to pick off high priority targets, eg necromancers. Another less loaded term might be like, "Priority rifle", since afaik that isn't a real earth weapon classification.

As I said in discord, I'm not married to a single one of its stats. As long as it can be used at a decent range to take out necromancers and masters, and ideally has some kind of setting to try to get a gib shot or otherwise take them down, it is doing its job

**Pull request merged**